### ISSUE TITLE
JSXFragment not supported

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#500]
# Language
JavaScript

# Sample input source code
https://github.com/piotrwitek/react-redux-typescript-guide/blob/master//playground/src/hooks/use-reducer.tsx
```
<>
      Count: {state.count}
      <button onClick={() => dispatch({ type: 'reset' })}>Reset</button>
      <button onClick={() => dispatch({ type: 'increment' })}>+</button>
      <button onClick={() => dispatch({ type: 'decrement' })}>-</button>
    </>
```

# Expected output
No message when in help mode

# Actual output (or error message)
```
 - JSXFragment - {"line":32,"column":4} - {"line":37,"column":7}
```

**Comment by k2snowman69** [comment]
Will release with 2.6.3 or greater

**Comment by k2snowman69** [comment]
Released with 3.0.0
